utils/parts.stanza

```diff
@@ -41,7 +43,13 @@ public defstruct ComponentCode :
   symbols: Tuple<SymbolCode>
   datasheet: String
   image: String
+  trust: String
+  mounting: String
+  case: String|UNKNOWN
+  dimensions: DimensionsCode
+  sourcing: SourcingCode
   metadata: Tuple<MetadataCode>
+  properties: Tuple<ComponentPropertyCode>
```

Comment:
Why are you duplicating the queryable fields with the parametric API into the esir data Code?